include/links-domain.php

```diff
@@ -99,11 +99,20 @@ public function get_hosts() {
 		$hosts = array();
 		foreach ( $this->options['domains'] as $lang => $domain ) {
 			$host = wp_parse_url( $domain, PHP_URL_HOST );
+
+			if ( ! is_string( $host ) ) {
+				continue;
+			}
+
 			// The function idn_to_ascii() is much faster than the WordPress method.
```

Comment:
Is it still the case since they made some changes?

Reply:
Probably, since the new version makes sure that the parameter is a string, so it adds processing.